Build/scripts/pgo/pre_pgo.cmd

```diff
@@ -12,6 +12,13 @@
 :: - build (using PGO profile)
 :: - post_pgo.cmd
 
+@echo off
+
+if "%PogoConfig%"=="False" (
+    echo "---- Not a Pogo Config. Skipping step."

```

Comment:
I did a grep for it, and all other instances of echo in cmd/bat files have no quotes. I'll make the change to be consistent.